### PR TITLE
[PAY-1070] Update TabSlider/SegmentedControl slider size on resize

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

- Updates the slider's selector to match the new sizes of each tab as they resize.
- Updates the slider to be accessible by using radio buttons/groups and labels for each option (you can now tab to it and select using the arrow keys!)
- Replaces usages of old `data-entry/TabSlider` with the Stems component `SegmentedControl`

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

Deletes old `TabSlider` component, opting for `SegmentedControl` instead.

Pretty wide reaching change, but I tested each area to ensure things still work and look as expected.

❗  **Only thing I noticed is font color on mobile web looks too light, and the theme slider on the settings page is slightly less tall?**

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Checked the following:
- Trending rewards modal on $AUDIO page
image

- Release type slider in UploadPage (TrackPreview)
image

- ImageSelectionPopup on UploadPage and CreatePlaylistModal
image
image

- NotificationsSettingsPage on **Mobile Web**
image

- NotificationSettings on Desktop
image

- Theme selector on **Mobile Web**
image

- Theme selector on Desktop
image

- EmbedModal
image

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

N/A

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

N/A

### CODE REVIEW

**Comment by dylanjeffers** [comment]
Will review tomorrow, but this looks epic!

File: packages/web/src/components/image-selection/ImageSelectionPopup.module.css [hunk 1]
```diff
@@ -7,7 +7,6 @@
 
 .slider {
   width: 100%;
-  padding: var(--unit-2) 3px;
```
Comment (rickyrombo) [inline packages/web/src/components/image-selection/ImageSelectionPopup.module.css:10]: no longer needed with SegmentedControl

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-theme-slider-resize

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-theme-slider-resize

**Review by Kyle-Shanks — APPROVED** [review]
clean 🧼

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-theme-slider-resize

**Pull request merged**